clients/core-java/pom.xml

```diff
@@ -28,6 +28,14 @@
   <name>Indy :: Client APIs :: Core for Java</name>
   
   <dependencies>
+    <dependency>
+      <groupId>org.commonjava.util</groupId>
+      <artifactId>o11yphant-honeycomb-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.commonjava.util</groupId>
```

Comment:
The core has many filter/reporter/cdi beans, etc. I worry about depending on them will be too much for client. I guess what you need is small part of core. I did a pr to separate them from core. pls review https://github.com/Commonjava/o11yphant/pull/51